src/components/Wearables/SelectorRow.tsx

```diff
@@ -56,22 +54,28 @@ export const SelectorRow: FC<SelectorRowProps> = (props) => {
   );
 };
 
-export const SelectorRowDefaultStyles = {
+const defaultStyles = createStyles('SelectorRow', (theme) => ({
   container: {
     minHeight: 40,
   },
   choiceText: {
     textAlign: 'right',
   },
   textWrapper: {
-    padding: Padding.medium,
+    padding: 12,
```

Comment:
Feel free to set this to something in the theme if something is close, even if it changes the UI slightly.  Or, if you deem this is best, that's fine too.

Reply:
I think it's generally better to set the margin and padding values for each element individually. I don't think setting these values globally at the theme level will work well everywhere.

Reply:
I've definitely seen "everything is based on the same theme value" go wrong / not scalable.  But also worth noting: manually entered values can't be affected when branding unless one does "component style overrides" which is a pretty advanced topic.  On the "general" side, I'd lean more towards using theme values in our default styles - and use manually entered values only where we know it's an exception that shouldn't change with a theme.  But, for example, text padding being controlled via 1 theme prop across an app seems like a good thing.  Can you explain more why you're thinking having more manual values in here is better?

Reply:
Good offline discussion about all of this, merging as is for now, but will be doing a follow up PR to address centralized spacing via the theme.